prow.go

```diff
@@ -834,13 +836,61 @@ func getClusterClient(m *jobManager, job *Job) (*BuildClusterClientConfig, error
 	return clusterClient, nil
 }
 
+func createAccessRBAC(namespace string, user string, client ctrlruntimeclient.Client) error {
+	roleBinding := &rbacapi.RoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "cluster-bot-user-access",
+			Namespace: namespace,
+		},
+		Subjects: []rbacapi.Subject{{Kind: "User", Name: user}},
+		RoleRef: rbacapi.RoleRef{
+			Kind: "ClusterRole",
+			Name: "admin",
+		},
+	}
+	if err := client.Create(context.Background(), roleBinding); err != nil && !errors.IsAlreadyExists(err) {
+		return err
+	}
+	return nil
+}
+
+func (m *jobManager) setupAccessRBAC(job *Job, namespace string) {
+	err := wait.ExponentialBackoff(wait.Backoff{Steps: 10, Duration: 2 * time.Second, Factor: 2}, func() (bool, error) {
+		clusterClient, err := getClusterClient(m, job)
+		if err != nil {
+			return false, err
+		}
+		if job.RequesterUserID != "" {
+			client, err := ctrlruntimeclient.New(clusterClient.CoreConfig, ctrlruntimeclient.Options{})
+			if err != nil {
+				return false, err
+			}
+			if err := createAccessRBAC(namespace, job.RequesterUserID, client); err != nil {
+				klog.Infof("could not create role binding for %s: %v", job.RequesterUserID, err)
```

Comment:
It's probably best if we dont start spraying folks RedHat ID's throughout our logs.  I'd recommend only printing their unique slack ID (`job.RequestedBy`).

Reply:
This particular error should probably be logged as an Errorf()

Reply:
Fixed